src/client/debugger/extension/adapter/factory.ts

```diff
@@ -142,8 +155,42 @@ export class DebugAdapterDescriptorFactory implements IDebugAdapterDescriptorFac
         return this.getExecutableCommand(interpreters[0]);
     }
 
+    private async showDeprecatedPythonMessage() {
+        const notificationPromptEnabled = this.persistentState.createGlobalPersistentState(
+            debugStateKeys.doNotShowAgain,
+            false,
+        );
+        if (notificationPromptEnabled.value) {
+            return;
+        }
+        const prompts = [Common.changePythonInterpreter, Common.doNotShowAgain];
+        const selection = await showErrorMessage(
+            localize(
+                'deprecatedDebuggerError',
```

Comment:
```suggestion
                'debug.deprecatedDebuggerError',
```
All other localizations are under the namespace.